frontend/src/metabase/querying/components/FilterValuePicker/SearchValuePicker/SearchValuePicker.tsx

```diff
@@ -35,19 +38,37 @@ export function SearchValuePicker({
   const [searchValue, setSearchValue] = useState("");
   const [searchQuery, setSearchQuery] = useState(searchValue);
 
-  const { data: fieldValues = initialFieldValues } = useSearchFieldValuesQuery(
+  const { data: searchFieldValues = initialFieldValues } =
+    useSearchFieldValuesQuery(
+      {
+        fieldId,
+        searchFieldId,
+        value: searchQuery,
+        limit: SEARCH_LIMIT,
+      },
+      {
+        skip: !searchQuery,
+      },
+    );
+
+  const { data: remappedFieldValues = [] } = useGetRemappedFieldValuesQuery(
     {
       fieldId,
-      searchFieldId,
-      value: searchQuery,
-      limit: SEARCH_LIMIT,
+      remappedFieldId: searchFieldId,
+      values: selectedValues,
     },
     {
-      skip: !searchQuery,
+      skip:
+        fieldId === searchFieldId ||
+        selectedValues.length === 0 ||
+        searchValue.length !== 0,
     },
   );
 
-  const options = useMemo(() => getFieldOptions(fieldValues), [fieldValues]);
+  const options = useMemo(
```

Comment:
Sometimes remapped values turn back to raw values.
In the video I'm using commas to apply subsequent values.


https://github.com/metabase/metabase/assets/6830683/ebfbda3a-bbfe-4d5c-8c0a-4a270b125ce5

Reply:
https://github.com/metabase/metabase/assets/6830683/d5786de2-fd50-43ea-b019-7704cbb3d956

Reply:
^ native inputs are not changed here, it's a different component

Reply:
@kamilmielnik the only reason I can think of why values can turn into raw values:
- search response -> pick value
- we can remap the value because search results are still here
- we send a new remapping request to the newly selected values
- you start typing something new
- there is a new search response
- if the search response becomes available before the remapping response, in this internal you would see unmapped values

Reply:
OK, here we go.